### PR TITLE
Clean Up GlCanvas

### PR DESCRIPTION
**Pull request by florian-kuebler**
Renamed and fixed clang-tidy warnings in GlCanvas

### CODE REVIEW

**Comment by beckerhe** [comment]
There seems to be a problem with the `draw_ui_` member variable.

I'm also mentioning @karupayun because I happen to know that he was also working on this code.

**Comment by beckerhe** [comment]
> There seems to be a problem with the `draw_ui_` member variable.
> 
> I'm also mentioning @karupayun because I happen to know that he was also working on this code.

Ahh. Too late. Race-condition with the force-push. :innocent:

File: OrbitGl/GlCanvas.cpp [hunk 1]
```diff
@@ -362,95 +318,75 @@ void GlCanvas::Render(int a_Width, int a_Height) {
...
-                       float a_MaxSize, bool a_RightJustified) {
-  m_TextRenderer.AddText(a_Text, a_X, a_Y, a_Z, a_Color, a_MaxSize, a_RightJustified);
-}
-
-int GlCanvas::AddText2D(const char* a_Text, int a_X, int a_Y, float a_Z, const Color& a_Color,
-                        float a_MaxSize, bool a_RightJustified, bool a_InvertY) {
-  return m_TextRenderer.AddText2D(a_Text, a_X, a_Y, a_Z, a_Color, a_MaxSize, a_RightJustified,
-                                  a_InvertY);
+void GlCanvas::AddText(const char* text, float x, float y, float z, const Color& color,
+                       float max_size, bool right_justified) {
```
Comment (karupayun) [inline OrbitGl/GlCanvas.cpp:376]: You probably must erase this function, because some previous change that I've made.

**Review by karupayun — APPROVED** [review]
Perfect Florian! :) 

Thanks for the big cleaning up!

**Review by beckerhe — APPROVED** [review]
Sorry for being late. Looks good to me. Thanks a lot!

**Pull request merged**